content/200-concepts/100-components/02-prisma-client/240-metrics.mdx

```diff
@@ -38,7 +38,7 @@ Prisma Client provides the following metrics:
   - `prisma_client_queries_wait`: The number of Prisma Client queries currently waiting for a connection because all connections are in use.
   - `prisma_pool_connections_busy`: The number of currently busy pool connections. These pool connections are currently executing a datasource query.
   - `prisma_pool_connections_idle`: The number of pool connections that are not currently being used. These pool connections are waiting for the next datasource query to run.
-  - `prisma_pool_connections_open`: The total number of [pool connections](/concepts/components/prisma-client/working-with-prismaclient/connection-pool#default-connection-pool-size) opened.
```

Comment:
This is still not the number of "opened", but currently "open".

Doesn't this description come from the API when using it?